python/nano/src/bigdl/nano/pytorch/lightning.py

```diff
@@ -56,30 +64,31 @@ def training_step(self, batch, batch_idx):
 
     def validation_step(self, batch, batch_idx):
         y_hat = self._forward(batch)
-        loss = self.loss(y_hat, batch[-1])  # use last output as target
-        self.log("val/loss", loss, on_epoch=True, prog_bar=True, logger=True)
+        if self.loss:
+            loss = self.loss(y_hat, batch[-1])  # use last output as target
+            self.log("val/loss", loss, on_epoch=True,
```

Comment:
Why this change? This PR should focus on OpenCV support; you may submit a separate PR for this change if needed.